### ISSUE TITLE
Style/StringLiterals: reports offense on \u0060 and \u0022

### ISSUE THREAD

**Issue by nijikon** [bbatsov/rubocop#3056]
## Expected behavior

Not to return error.

## Actual behavior

```ruby
$ rubocop text.rb
Inspecting 1 file
C

Offenses:

text.rb:1:25: C: Prefer single-quoted strings when you don't need string interpolation or special symbols.
OpenStruct.new(content: "\u0060")
                        ^^^^^^^^
text.rb:3:25: C: Prefer single-quoted strings when you don't need string interpolation or special symbols.
OpenStruct.new(content: "\u0022")
                        ^^^^^^^^

1 file inspected, 2 offenses detected
```
## Steps to reproduce the problem
```ruby
$ cat text.rb
OpenStruct.new(content: "\u0060")
OpenStruct.new(content: "\u00b4")
OpenStruct.new(content: "\u0022")
OpenStruct.new(content: "\u201d")
```

## RuboCop version
```
$ rubocop -V
0.39.0 (using Parser 2.3.0.7, running on ruby 2.3.0 x86_64-darwin15)
```

**Comment by mikegee** [comment]
I guess the cop is trying to tell you to switch those two. Autocorrect changes the two that it can:

```bash
rubocop master % ./bin/rubocop text.rb --only Style/StringLiterals --auto-correct
Inspecting 1 file
C

Offenses:

text.rb:1:25: C: [Corrected] Prefer single-quoted strings when you don't need string interpolation or special symbols.
OpenStruct.new(content: "\u0060")
                        ^^^^^^^^
text.rb:3:25: C: [Corrected] Prefer single-quoted strings when you don't need string interpolation or special symbols.
OpenStruct.new(content: "\u0022")
                        ^^^^^^^^

1 file inspected, 2 offenses detected, 2 offenses corrected
rubocop master % cat text.rb
OpenStruct.new(content: '`')
OpenStruct.new(content: "\u00b4")
OpenStruct.new(content: '"')
OpenStruct.new(content: "\u201d")
```

**Comment by alexdowad** [comment]
> Now I'm surprised the other two weren't changed.

The other 2 are not ASCII.

This is expected behavior. I suggest this issue can be closed.
